cha-ching-app/src/App.js

```diff
@@ -1,28 +1,35 @@
 import { Elements, PaymentElement } from "@stripe/react-stripe-js";
 import { loadStripe } from "@stripe/stripe-js";
+import { useState } from "react";
 import "./App.css";
 
 const stripePromise = loadStripe(process.env.REACT_APP_PUBLIC_STRIPE_KEY);
 
 function App() {
-  const options = {
-    clientSecret: "insert client secret here please",
-  };
+  const [options, setOptions] = useState({});
 
-  function GetClientSecret() {
+  const isEmpty = (object) => Object.keys(object).length === 0;
+
+  async function GetClientSecret() {
```

Comment:
Usually functions are written in camel case, starting with a lowercase letter.

Reply:
I also see there's an async here but currently we're not awaiting on anything so it can be removed for now